### PR TITLE
Implement update dialog design

### PR DESCRIPTION
**Pull request by Naatan**
https://www.pivotaltracker.com/story/show/177827479

### CODE REVIEW

File: activestate.yaml [hunk 1]
```diff
@@ -126,8 +140,42 @@ scripts:
...
+    description: Builds the State Tool update dialog app's frontend. Be sure to run build-updlg-assets first if you changed image assets.
+    language: bash
+    value: |
+      set -e
+      echo "Building frontend. If you updated img assets be sure to first run build-updlg-assets"
+      pushd cmd/state-update-dialog/frontend
+      sassc scss/main.scss generated/main.css
+      npm run build
+      popd
+  - name: build-updlg-assets
```
Comment (daved) [inline activestate.yaml:157]: Maybe `preprocess-updlg`

File: cmd/state-update-dialog/frontend/main.html [hunk 2]
```diff
@@ -3,8 +3,40 @@
...
+        <div class="row" id="warning-wrapper">
+            <div class="column">
+                <strong class="red">WARNING:</strong> <span id="warning"></span>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column buttons">
+                <button class="button button-outline push-right" id="close-btn">Close</button>
+                <button class="button">Update</button>
+                <!--<button class="button button-outline">Remind Me Later</button>-->
```
Comment (daved) [inline cmd/state-update-dialog/frontend/main.html:37]: Is this useful?

File: cmd/state-update-dialog/frontend/main.js [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+
+    populateChangelog();
+
+    document.getElementById("close-btn").addEventListener("click", () => backend.Bindings.Exit());
+}
+
+function populateChangelog(tries) {
+    tries = tries || 0;
+    if (tries > 10) {
+        return;
```
Comment (daved) [inline cmd/state-update-dialog/frontend/main.js:32]: I'm probably misunderstanding, but, at a glance, I think this should be `if (tries > 10) { tries = 10 }`

**Review by daved — APPROVED** [review]

**Comment by daved** [comment]
A handful of files that are useful to review were hidden from me by GitHub. I did take a look at them by checking out the branch locally.

File: cmd/state-update-dialog/frontend/main.js (hunk 3 above)
Comment (Naatan) [reply cmd/state-update-dialog/frontend/main.js:32]: No, I don't want to try more than 10 times.

**Pull request merged**